.dockerignore

```diff
@@ -2,6 +2,9 @@
 # Ignore everything
 **
 
+!shared
```

Comment:
i'm not sure if this is needed or not.

the rust other rust components only need to expose 
```
!target/<arch1>/debug/<component>
!target/<arch1>/release/<component>
!target/<arch2>/debug/<component>
!target/<arch2>/release/<component>
```

generally, that's because the rust components don't use docker build to create the binaries ... they copy the components built by the cross-build Action.  i see you using the actions, so it seems like your component should be created.  you may be able to delete these lines and have your Dockerfile just copy the binaries (rather than build them).

Reply:
Thanks! I'll try it... I've now broken the Dockerfile (for ARM32) too

Reply:
Ugh, now I understand... you `cargo build` and **then** containerize the ./target/release`